README.md

```diff
@@ -73,7 +73,7 @@ For protobuf-based codegen integrated with the Maven build system, you can use
     <extension>
       <groupId>kr.motd.maven</groupId>
       <artifactId>os-maven-plugin</artifactId>
-      <version>1.4.1.Final</version>
+      <version>1.5.0.Final</version>
```

Comment:
Bumping the version here is fine to do. It doesn't look like it has any bug fix, just a new feature. But staying current is useful.